src/utils/index.ts

```diff
@@ -239,6 +241,8 @@ export const isEnabledOGMessage = (message: UserMessage): boolean => (
 );
 
 export const getUIKitMessageTypes = (): UIKitMessageTypes => ({ ...UIKitMessageTypes });
+
+// Do not use this for MultipleFilesMessage. Use isMultipleFilesMessage() instead.
```

Comment:
```suggestion
/**
 * Do not use this for MultipleFilesMessage. Use isMultipleFilesMessage() instead.
 */
```
nit;
To give a hint in IDE, the comment format should be like the above. 
![Screenshot 2023-09-12 at 5 55 13 PM](https://github.com/sendbird/sendbird-uikit-react/assets/10060731/10f0b2bb-1c4e-4c12-b34b-c11fe19a983f)

Reply:
nice!